spa/src/pages/ConfigSteps/index.tsx

```diff
@@ -257,7 +254,15 @@ const ConfigSteps = () => {
 			}
 			setLoggedInUser(OAuthManager.getUserDetails().username);
 			setLoaderForLogin(false);
-			await getOrganizations();
+			setOrganizations([]);
+			if (status === true) {
+				const result = await getOrganizations();
+				if (result.success) {
```

Comment:
Can combine them instead of the nested ifs
```suggestion
				if (result.success && result.orgs.length === 0) {
```